include/tss2/tss2_tctildr.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: BSD-2
```

Comment:
The correct SPDX should be `BSD-2-Clause`

Also all others are a little different, but that's minor and I don't care too much...
```
/* SPDX-License-Identifier: BSD-2-Clause */
/***********************************************************************;
 * Copyright (c) 2015-2018, Intel Corporation
 *
 * All rights reserved.
 ***********************************************************************/
```
But the la

Reply:
Good catch w/r/t the SPDX identifier. Just pushed fixes for the one you caught and 2 others.

Reply:
w/r/t the file header / comment block: I'm trying to cut down on the number of characters here. We should be working to eliminate these blocks of comments completely (they have no meaning according to @williamcroberts  in #1445). Not trying to do that here but I drop those long strings of '*'s when I remembered to. My preference is to leave this as-is.

Reply:
Yeah just kill these, no need to introduce these if we don't have to.

Reply:
Just to be clear for when I'm doing the python bindings, we don't need these right (I'm gathering from these comments), but why the shift? (I've always thought it was re-redundant but just curious)

Reply:
You actually don't need any of it, as the LICENSE file indicates the LICENSE for all files. However, having the SPDX identifier is no big deal, its the copyright lines you don't want to add.